swift-android.patch

```diff
@@ -16,20 +54,32 @@ index 9932b854cb6..ad3ac757665 100644
          options.define('CMAKE_ANDROID_NDK:PATH', args.android_ndk)
          return options
  
+diff --git a/swift-corelibs-foundation/CMakeLists.txt b/swift-corelibs-foundation/CMakeLists.txt
+index 7f290d16..95366592 100644
+--- a/swift-corelibs-foundation/CMakeLists.txt
++++ b/swift-corelibs-foundation/CMakeLists.txt
+@@ -55,6 +55,7 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+ set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
++set(CMAKE_SHARED_MODULE_CREATE_Swift_FLAGS "")
```

Comment:
Sigh, I built fine locally with CMake 3.28, but the github image has 3.30 installed. I downloaded that CMake locally to reproduce the CI failure, and this patch got that local build working again, but it doesn't fix the CI! 😡

Going to leave it alone till later today: anyone else want to look into it, be my guest.

Reply:
Never mind, this doesn't work locally either: I had used it with CMake 3.28 by mistake, which already works.